tracks/julia/exercises/leap/mentoring.md

```diff
@@ -0,0 +1,17 @@
+### Mentor guidance
+
+We don't need to bikeshed on this too long. Just prompt them to improve their code if they do something needlessly convoluted. `if` statements are okay, but consider prompting them with the first or third suggestions.
+
+
+### Common suggestions
+
+(Formatted so you can copy and paste easily)
+
+- Whenever you have an expression like ``if condition; return true; else; return false``, you can rewrite it as ``return condition``.
+- Obviously you can do as you please, but it is conventional to have one space either side of each operator: ``year % 400 == 0``.
+- Consider writing a truth table for this exercise and see if you can work out what the simplest predicates are and how they could be combined with ``&&`` and ``||``
```

Comment:
Could you add a link to a simple example truth table? Programmers without formal CS/maths background might not be familiar with it.

Reply:
Done